x/distribution/module.go

```diff
@@ -161,7 +161,7 @@ func (AppModule) ConsensusVersion() uint64 { return ConsensusVersion }
 
 // BeginBlock returns the begin blocker for the distribution module.
 func (am AppModule) BeginBlock(ctx context.Context) error {
-	c := sdk.UnwrapSDKContext(ctx)
+	c := sdk.UnwrapSDKContext(ctx) // TODO: remove and use context.Context after including the comet service
```

Comment:
is there an issue for that?

Reply:
opened one: https://github.com/cosmos/cosmos-sdk/issues/19565